CONTRIBUTING.md

```diff
@@ -1,7 +1,72 @@
+# Thank you for considering to contribute!
+
+Contributions are always welcome. We accept [bug fixes](#bugs),
+[proposals and new ideas](#proposals-for-new-content), or simply [editorial
+improvements](#editorial-improvements).
+
+By submitting a pull request you are implicitly adhering to the
+[contribution agreement](#contribution-agreement)
+
+## Bugs
+
+You can report a bug by creating an
+[issue](https://github.com/ARM-software/acle/issues). You can also fix
+it yourself with a [pull
+request](https://github.com/ARM-software/acle/pulls).
+
+## Proposals for new content
+
+We are open to any proposal, including the introduction of new sets of
+intrinsics.
+
+For adding new intrinsics or preprocessor macros, we recommend
+presenting them in a [pull
+request](https://github.com/ARM-software/acle/pulls), together with
+the following helpful information:
+
+1. A rationale covering why these intrinsics or macros.
```

Comment:
```suggestion
1. A rationale in favour of these intrinsics or macros.
```

Reply:
My suggested edit might not be ideal -- but 'A rationale why these X' reads as though some words are missing. In speech you can put emphasis on 'why', but it doesn't work as well in writing.